src/App.js

```diff
@@ -1,91 +1,128 @@
 import React from 'react';
-import InputBar from './components/InputBar';
-import TodoBox from './components/TodoBox';
+
 import Tabs from './components/Tabs';
+import InputBar from './components/InputBar';
+import TodoList from './components/TodoList';
+
 import './assets/css/';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
 
-    const storageData = window.localStorage.getItem('todoData');
-    const todos = storageData ? JSON.parse(storageData) : [];
-
     this.state = {
-      todos: todos,
+      todos: [],
       tab: -1,
       edit: null,
-      editIndex: null
+      editIndex: null,
+      search: '',
     };
 
     this.startEdit = false;
```

Comment:
Why startEdit is not a state here?

Reply:
It was only used to check if edit object has been be send to render and set to it to false immediately so that when input field is typed its value is updated.